src/components/Match/AdditionalSuggestions/AdditionalSuggestionsScroll.js

```diff
@@ -116,7 +117,11 @@ const AdditionalSuggestionsScroll = ( {
 
   return (
     <View className="mt-4 mb-7">
-      <Heading3 className="mx-5 mb-3">{t( "It-might-also-be" )}</Heading3>
+      <Heading3 className="mx-5 mb-3">
```

Comment:
Also looks like the styling is off a touch here. In Figma there are 20px above this text and 15px below, while here it's 12px below and I believe 24px above (mt-4 is 16px and there's all the pt-2 of 8px on the parent View of this component).